src/components/ProductList.jsx

```
@@ -19,14 +19,38 @@ function ProductList() {
     // TODO: POST API를 호출하고 fetchProducts() 호출
+    try {
+      await fetch("${API_URL}/${newProduct}", {
+        method: "POST",
+      });
+      fetchProducts();
+    } catch (error) {
+      console.error("Error:", error);
+    }
   };
 
   const handleEdit = async (updatedProduct) => {
```

ESLint

[error] 'updatedProduct' is defined but never used.

(no-unused-vars)